coreblocks/func_blocks/fu/lsu/dummyLsu.py

```diff
@@ -228,17 +228,18 @@ def elaborate(self, platform):
         m.submodules.requester = requester = LSURequester(self.gen_params, self.bus)
 
         m.submodules.requests = requests = Forwarder(self.fu_layouts.issue)
-        m.submodules.results = results = self.forwarder = FIFO(self.lsu_layouts.accept, 2)
+        m.submodules.results_noop = results_noop = self.forwarder = FIFO(self.lsu_layouts.accept, 2)
```

Comment:
```suggestion
        m.submodules.results_noop = results_noop = FIFO(self.lsu_layouts.accept, 2)
```
`self.forwarder = FIFO(` :) ; `self.forwarder` is also unused

Reply:
Not here, I missed the separate PR